Resources/Prototypes/_Backmen/Reagents/Materials/materials.yml

```
@@ -5,3 +5,11 @@
   color: "#53a9ff"
   stackEntity: MaterialBluespace
   price: 15
+
+- type: material
+  id: NormalityCrystal
+  name: normality сrystal
+  icon: { sprite: Backmen/Objects/Materials/materials.rsi, state: normalitycrystal }
+  color: "#77e5f6"
+  stackEntity: NormalityCrystal
+  price: 20
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)